bBridgeAPISDK/Common/HttpRequester.cs

```diff
@@ -3,32 +3,49 @@
 using System.Net.Http;
 using System.Text;
 using System.Threading.Tasks;
-using bBridgeAPISDK.Authorization.Interfaces;
-using bBridgeAPISDK.Utils.Interfaces;
+using bBridgeAPISDK.Common.Authorization.Interfaces;
+using bBridgeAPISDK.Common.Interfaces;
 using Newtonsoft.Json;
 
-namespace bBridgeAPISDK.Utils
+namespace bBridgeAPISDK.Common
 {
     public class HttpRequester : IAsyncHttpRequester
     {
         private readonly IAuthorizer authorizer;
-        public HttpRequester(Uri baseAdress, IAuthorizer authorizer = null)
+        /// <summary>
+        /// Constructor of basic http requester
+        /// </summary>
+        /// <param name="baseAdress">Base API adress</param>
+        /// <param name="authorizer">Authorizer of a user (if needed)</param>
+        public HttpRequester(string baseAdress, IAuthorizer authorizer = null)
         {
             this.authorizer = authorizer;
             BaseAdress = baseAdress;
         }
-        public Uri BaseAdress { get; }
+        /// <summary>
+        /// Based API URI
+        /// </summary>
+        public string BaseAdress { get; }
 
+        /// <summary>
+        /// Makes async http request in POST if data specified and GET if data is not specified
+        /// </summary>
+        /// <typeparam name="T">Result deserialization type</typeparam>
+        /// <param name="urlSuffix">Suffix of the aprticular URL request</param>
+        /// <param name="data">Data to be sent in POST request body</param>
+        /// <returns>Request result deserialized as T</returns>
         public async Task<T> RequestAsync<T>(string urlSuffix, object data = null) where T : class
         {
             using (var client = new HttpClient())
```

Comment:
creation of new client for each request may be inefficient, for example, because http clients usually have a connection pool to speed up multiple requests